src/renderers/webxr/WebXRManager.js

```diff
@@ -503,11 +503,8 @@ class WebXRManager extends EventDispatcher {
 
 			// update user camera and its children
 
-			camera.position.copy( cameraVR.position );
-			camera.quaternion.copy( cameraVR.quaternion );
-			camera.scale.copy( cameraVR.scale );
 			camera.matrix.copy( cameraVR.matrix );
-			camera.matrixWorld.copy( cameraVR.matrixWorld );
+			camera.matrix.decompose( camera.position, camera.quaternion, camera.scale );
```

Comment:
Is  there any reason doing a second decompose is better than copying the results like it was before? I think decompose is a bit expensive, so probably best not to call it more than it needs to. 

Then the only difference would be not copying the `matrixWorld`. I don't remember the details of how three decides when to update that or not. @mrdoob do you see  any  trouble with this?

Reply:
The subtle difference is between local and global space.

The old method of:
```javascript
cameraVR.matrixWorld.decompose( cameraVR.position, cameraVR.quaternion, cameraVR.scale );
camera.position.copy( cameraVR.position );
camera.quaternion.copy( cameraVR.quaternion );
camera.scale.copy( cameraVR.scale );
```
is copying **world**Matrix values into **local** position/rotation/scale variables.

This broke any VR application that had the VRCamera as the child of another transform.